scripts/download_backtest_data.py

```diff
@@ -60,55 +70,59 @@
         }
     }
 
+configuration._load_logging_config(config)
+configuration._load_datadir_config(config)
 
-dl_path = Path(DEFAULT_DL_PATH).joinpath(config['exchange']['name'])
-if args.export:
-    dl_path = Path(args.export)
-
-if not dl_path.is_dir():
-    sys.exit(f'Directory {dl_path} does not exist.')
+dl_path = Path(config['datadir'])
 
 pairs_file = Path(args.pairs_file) if args.pairs_file else dl_path.joinpath('pairs.json')
-if not pairs_file.exists():
-    sys.exit(f'No pairs file found with path {pairs_file}.')
 
-with pairs_file.open() as file:
-    PAIRS = list(set(json.load(file)))
+if not pairs or args.pairs_file:
```

Comment:
Hmm, it should override -- if `--pairs-file` is specified, the pairs file should be used instead of the whitelist from the config... I guess I tested it. Do you obtain diff. behaviour?

`scripts/download_backtest_data.py --config config.json --exchange binance --pairs-file user_data/data/binance/pairs.json` -- uses pairs file and download data for binance instead of bittrex, the rest of settings is taken from config.json.